capsules/src/bme280.rs

```diff
@@ -304,28 +308,28 @@ impl<'a> I2CClient for Bme280<'a> {
                             return;
                         }
 
-                        let var1 = (((adc_temperature >> 3) - ((calib.temp1 as usize) << 1))
-                            * (calib.temp2 as usize))
+                        let var1 = (((adc_temperature >> 3) - ((calib.temp1 as i32) << 1))
+                            * (calib.temp2 as i32))
                             >> 11;
-                        let var2 = (((((adc_temperature >> 4) - (calib.temp1 as usize))
-                            * ((adc_temperature >> 4) - (calib.temp1 as usize)))
+                        let var2 = (((((adc_temperature >> 4) - (calib.temp1 as i32))
+                            * ((adc_temperature >> 4) - (calib.temp1 as i32)))
                             >> 12)
-                            * (calib.temp3 as usize))
+                            * (calib.temp3 as i32))
                             >> 14;
 
                         self.t_fine.set(var1 + var2);
 
-                        let temperature = ((self.t_fine.get() * 5 + 128) >> 8) / 100;
+                        let temperature = (self.t_fine.get() * 5 + 128) >> 8;
```

Comment:
We still need the divide by 100

Reply:
Wouldn't dividing by 100 result in temperature in degrees Celsius? Every other temperature sensor returns hundredths of degrees Celsius instead. Am I misunderstanding something?